rootfs/rootfs/usr/local/etc/init.d/docker

```diff
@@ -116,7 +116,7 @@ restart() {
 }
 
 check() {
-    [ -f /var/run/docker.pid ] && ps -o pid | grep "^\s*$(cat /var/run/docker.pid)$" > /dev/null 2>&1
+    [ -f /var/run/docker.pid ] && ps -A -o pid | grep `cat /var/run/docker.pid` > /dev/null 2>&1
```

Comment:
Why the switch to backticks?  I personally prefer to avoid the backtick form since it's so easy to make mistakes with (and can't be nested).  Is it the `-A` that fixed it, or the `grep` regex changes?  As-is, if our PID from the file is `123` and a PID of `9123` is running, this will match when it shouldn't.